Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:buster
+
+# install apt dependencies
+RUN apt-get update -y
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y locales chromium=90.0.4430.212-1~deb10u1 chromium-driver=90.0.4430.212-1~deb10u1 python3 python3-pip python-websockify git xorg vnc4server autocutsel lxde-core novnc
+RUN pip3 install --upgrade pip
```

Comment:
Wieso muss man `pip` installieren, wenn pip offensichtlich schon installiert ist?

Reply:
Sonst kommt dieser Fehler wegen der zu alten Version von pip im buster repo.

`Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-etn4pwuf/PyQt5/`